query-engine/driver-adapters/js/adapter-pg/src/conversion.ts

```diff
@@ -3,6 +3,31 @@ import { types } from 'pg'
 
 const PgColumnType = types.builtins
 
+/**
```

Comment:
TODO: extract the duplicate code into a new common package that both pg and neon driver adapters will depend on (`@prisma/driver-adapter-utils-postgres` or something like that)